ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-master.yaml

```diff
@@ -34,6 +34,27 @@ tests:
           cpu: 200m
           memory: 400Mi
     workflow: ipi-gcp
+- as: go-funtests
```

Comment:
Can you keep the naming convention similar to other tests?
For example: `e2e-azure-go-tests`.

Reply:
corrected